python/DYNAMODB_TABLE_ENCRYPTED_KMS/DYNAMODB_TABLE_ENCRYPTED_KMS.py

```diff
@@ -0,0 +1,57 @@
+'''
+#####################################
+##           Gherkin               ##
+#####################################
+Rule Name:
+  DYNAMODB_TABLE_ENCRYPTED_KMS
+
+Description:
+  Check whether DynamoDB Table is encrypted with KMS.
+
+Rationale:
+  Encrypting on Dynamo ensure that no data is written on disk in clear text.
```

Comment:
Need to AWS Marketing read. Encrypting Amazon DynmoDB tale ensure that...

Reply:
changes are completed